targets/esp32/mqtt/Makefile

```diff
@@ -0,0 +1,18 @@
+all:
+	platformio -f  run
+
+upload:
+	platformio -f  run --target upload
+
+clean:
+	platformio -f  run --target clean
+
+program:
+	platformio -f  run --target program
+
+uploadfs:
+	platformio -f  run --target uploadfs
+
```

Comment:
Since `platformio -f  run --target` is being repeated can you move to something like
```make
define platformio
	platformio -f  run --target $(1)
endef

run-%:
	$(call platformio,$*)
```